2021-03-18-if-else-exercises/main.js

```diff
@@ -0,0 +1,61 @@
+/* 
+    Exercise 1:
+    - complete the if/else statement so that the result is true if the input number is bigger equal than 70 and smaller equal than 99 
+    - try changing the input and check if it's still correct
+*/
+
+let inputEx1 = 5;
+
+if () {
+    console.log("#1 Is " + inputEx1 + "between 70 and 99?", true);
+} else {
+    console.log("#1 Is " + inputEx1 + "between 70 and 99?", false);
+}
+
+/*
+    Exercise 2:
```

Comment:
I wonder whether we should tell a little story around it to make it more "real life"? E.g. "You get the sales number of your two offices, and your boss wants to know whether one of them sold between 70 and 900 units."

Reply:
very nice idea, I'll change them to this